app/Party.php

```diff
@@ -705,16 +705,21 @@ public function devices()
 
     public function hoursVolunteered()
```

Comment:
Don't have to add one in now, but as a side-note, this to me is the kind of thing that benefits from a pure unit test.

Reply:
i.e. something that is purely business logic.  And that could also do with a little documentation/reassurance around some of the assumptions built in to it.